doc/user-guide/io.rst

```diff
@@ -519,7 +519,7 @@ the ability to store and analyze datasets far too large fit onto disk
 Xarray can't open just any zarr dataset, because xarray requires special
 metadata (attributes) describing the dataset dimensions and coordinates.
 At this time, xarray can only open zarr datasets that have been written by
-xarray. For implementation details, see :ref:`zarr_encoding`.
+xarray or `NCZarr`_. For implementation details, see :ref:`zarr_encoding`.
```

Comment:
I might say "netCDF" here instead of "NCZarr".

NCZarr is the format, but NetCDF is the name of the software that writes it.

Reply:
Right! GDAL and probably other libraries create xarray-compatible zarr datasets as well, so I've edited a little that sentence.